### PR TITLE
Swik 1171 more user information when searching

### PR DESCRIPTION
**Pull request by TBoonX**
This have to be merged with the platform branch (same name).
I changed the search user route code, which now also returns the description and the country of a user. Also two small improvements of the code in this area.

### CODE REVIEW

File: application/controllers/handler.js [hunk 1]
```diff
@@ -422,30 +422,43 @@ module.exports = {
...
       .then((cursor2) => cursor2.limit(10))
       .then((cursor3) => cursor3.toArray())
       .then((array) => {
         // console.log('handler: checkUsername: similar usernames', array);
         let data = array.reduce((prev, curr) => {
+          let description = curr.username;
+          if (curr.organization)
+            description = description + ', ' + curr.organization;
+          if (curr.country)
+            description = description + ', ' + curr.country;
```
Comment (rmeissn) [inline application/controllers/handler.js:445]: This looks a little a strange, but I haven't tested the route yet. Is there a reason you're constructing a description?

**Review by rmeissn — COMMENTED** [review]
Code looks perfect, will test the functionality now

File: application/controllers/handler.js (hunk 1 above)
Comment (rmeissn) [reply application/controllers/handler.js:445]: Makes sense now that I tested it.

**Comment by rmeissn** [comment]
Code and functionality for the user-service seems to be fine. Haven't tested the UI yet.

**Comment by kadevgraaf** [comment]
I will merge because this is fine and convenient to test the UI :)

**Pull request merged**